src/main/webapp/resources/celJS/celTabMenu/loadTinyMCE-async.js

```diff
@@ -116,6 +117,9 @@
     console.log('getUninitializedMceEditors: start ', mceParentElem);
     var mceEditorsToInit = new Array();
     $(mceParentElem).select('textarea.mceEditor').each(function(editorArea) {
+      if (!editorArea.id || (editorArea.id === '')) {
```

Comment:
the case of the strict empty check is already treated in the preceding truthy check, hence:
```suggestion
      if (!editorArea.id) {
```

Reply:
sorry, zu spät gesehen :-(